lib/tests/2wp.js

```diff
@@ -777,6 +777,47 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should reject and refund a pegout when fee per kb is above value', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+      const initialBtcRecipientAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      const initialRskSenderBalanceInWeisBN = await rskTxHelper.getBalance(senderRecipientInfo.rskRecipientRskAddressInfo.address);
+      const pegoutValueInRbtc = MINIMUM_PEGOUT_AMOUNT_IN_RBTC;
+
+      // The minimum pegout value plus 1 satoshi to make the fee per kb exactly above the pegout value.
+      const newFeePerKbInSatoshis = btcToSatoshis(MINIMUM_PEGOUT_AMOUNT_IN_RBTC) + 1;
```

Comment:
According to the logic in [rskj](https://github.com/rsksmart/rskj/blob/8c1e3a055cb11a2738e47122fa3f96fe3bba8fc0/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L880), setting the fee per kb to the minimum peg-out + 1 is way above the peg-out value.

I think you should use `getEstimatedFeesForNextPegOutEvent` to check if the new fee set really makes the estimated fees above the sent amount.